tasks/setup-Debian.yml

```diff
@@ -12,6 +12,7 @@
       - apt-transport-https
       - ca-certificates
     state: present
+    update-cache: true
```

Comment:
I would also suggest adding `cache_valid_time: 3600` or some other value so that this doesn't update the cache every single time it's run (especially in a small period of time).

Reply:
absolutely. One hour seems ok. Will update